src/Cassandra/UdtMap.cs

```diff
@@ -83,7 +84,9 @@ public abstract class UdtMap
         protected readonly Dictionary<PropertyInfo, string> _propertyToFieldName;
         // ReSharper enable InconsistentNaming
         private Serializer _serializer;
-        protected const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+        public const BindingFlags PropertyFlags = BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Public | BindingFlags.Instance;
+
+        public TypeConverter TypeConverter { get; private set; }
```

Comment:
You could use a single static instance of the default type converter as a internal field:

```csharp
    public class UdtMap
    {
         internal static TypeConverter TypeConverter = new DefaultTypeConverter();

         // Use it in the ToObject() method
         internal object ToObject(object[] values)
         {
              // ...
              TypeConverter.ConvertDbValue(/*...*/);
              // ...
          }
    }

    // Reuse the same instance in the UdtSerializer class
    public class UdtSerializer
    {
         public override byte[] Serialize(/*...*/)
         {
              // ...
              fieldValue = UdtMap.TypeConverter.ConvertObjectValue(/**/);
              // ...
          }
    } 
```

Avoid referring to `MappingConfiguration` as it's from the Mapper component.